coroner-client/src/main/java/backtraceio/coroner/response/CoronerResponseGroup.java

```diff
@@ -0,0 +1,25 @@
+package backtraceio.coroner.response;
+
+import java.util.List;
+
+public class CoronerResponseGroup {
+    private static final Integer EXPECTED_NUMBER_OF_EL = 3;
+    @SuppressWarnings("unused")
+    private final String groupIdentifier;
+    private final List<Object> values;
+
+    public CoronerResponseGroup(final List<Object> obj) throws IllegalArgumentException {
+        if (obj == null || obj.size() != EXPECTED_NUMBER_OF_EL) {
```

Comment:
what if some attributes already exists, or we have less attributes than 3 (which i believe is fine)

Reply:
Here there is a check if in the response below, element of values arrays contains 3 elements inside, in this case "*", array of attributes and 818. It's not a check if in the attributes array we have less than 3 elements. It's part of checking correctness of the response format.

```
"values": [
      [
        "*",
        [
          [
            "DumpWithoutCrash"
          ],
          [
            "{\"frame\":[\"libbacktrace-native.so\",\"libbacktrace-native.so\",\"libart.so\",\"libc.so\",\"libbacktrace-native.so\",\"libbacktrace-native.so\",\"libart.so\",\"libbacktrace-native.so\",\"libart.so\",\"libart.so\",\"libart.so\",\"libart.so\",\"libc.so\",\"libart.so\",\"libart.so\",\"libart.so\",\"libart.so\",\"libart.so\",\"libart.so\",\"libart.so\",\"libart.so\",\"libart.so\",\"libart.so\",\"libart.so\",\"libart.so\",\"libart.so\",\"libart.so\",\"libart.so\",\"libart.so\"]}"
          ],
          [
            "71baf2e3-0fe3-48fa-ba18-d2b3d98d1e75"
          ],
          [
            "exception"
          ]
        ],
        818
      ]
    ],
```